src/main/java/org/icij/datashare/Neo4jResource.java

```diff
@@ -475,27 +519,25 @@ protected InputStream dumpGraph(
             request);
         checkExtensionProject(projectId);
         checkNeo4jAppStarted();
-        String database = neo4jProjectDatabase(projectId);
-        return client.dumpGraph(database, neo4jAppRequest);
+        return client.dumpGraph(projectId, neo4jAppRequest);
     }
 
     protected InputStream sortedDumpGraph(
         String projectId, org.icij.datashare.Objects.SortedDumpRequest request
     ) throws URISyntaxException, IOException, InterruptedException {
         checkExtensionProject(projectId);
         checkNeo4jAppStarted();
-        String database = neo4jProjectDatabase(projectId);
         Statement statement = request.query.defaultQueryStatement(getDocumentNodesLimit());
         org.icij.datashare.Objects.Neo4jAppDumpRequest neo4jAppRequest =
             new org.icij.datashare.Objects.Neo4jAppDumpRequest(
                 request.format, statement.getCypher()
             );
-        return client.dumpGraph(database, neo4jAppRequest);
+        return client.dumpGraph(projectId, neo4jAppRequest);
     }
 
     private <T> Payload wrapNeo4jAppCall(Callable<T> responseProvider) {
         try {
-            return new Payload(responseProvider.call()).withCode(200);
+            return new Payload(responseProvider.call());
         } catch (InvalidProjectError e) {
             return new Payload("application/problem+json", e).withCode(403);
```

Comment:
I just learned about the `application/problem+json` media type , https://datatracker.ietf.org/doc/html/rfc7807#autoid-4 thanks